Components/ReusableComponent/functions.php

```diff
@@ -31,6 +32,17 @@ function getACFLayout()
                 'return_format' => 'id',
                 'instructions' => 'Add or edit <a href="/wp/wp-admin/edit.php?post_type=reusable-component">reusable components</a>.'
             ],
-        ]
+        ],
     ];
 }
+
+add_filter('acf/prepare_field/name=reusableId', function ($field) {
+    // Update instructions with selected post link
+    if ($field['value']) {
+        $postTitle = get_the_title($field['value']);
+        $instructions = $field['instructions'];
+        $instructions = 'Edit <a class="reusable-postLink" href="/wp/wp-admin/post.php?post=' . $field['value'] . '&action=edit&classic-editor" target="_blank" rel="noopener noreferrer">' . $postTitle . '</a>.';
```

Comment:
Looks like the first `$instructions` definition isn't needed, as it's overwritten right after.

Do we also need the parameter `&classic-editor` in the link?